pandas/tests/extension/decimal/array.py

```diff
@@ -101,10 +101,8 @@ def take(self, indexer, allow_fill=False, fill_value=None):
                       allow_fill=allow_fill)
         return self._from_sequence(result)
 
-    def copy(self, deep=False):
```

Comment:
can you add a test in the pandas/tests/extension/constructors.py so it hits all EA's (for view & copy)